test/FunctionalTests/Balancer/LeastUsedBalancerTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+#if SUPPORT_LOAD_BALANCING
+#if NET5_0_OR_GREATER
```

Comment:
Do we need this check here if `SUPPORT_LOAD_BALANCING` is only set when the `TargetFramework` is greater than `net5.0`?